UPDATING.md

```diff
@@ -22,16 +22,33 @@ under the License.
 This file documents any backwards-incompatible changes in Superset and
 assists people when migrating to a new version.
 
-## 2.0.1
+## Next
```

Comment:
we shouldn't have a next in 2.0, but we'll have this in master. I think you'll want to create two different PRs.

Reply:
The one for master was already created and merged :)

Reply:
Ok already merged one with next into master.  Should I just deleted next here?